docs/using-pre-commit.md

```diff
@@ -27,35 +27,41 @@ Note that these hooks do not update the `.ipynb` notebook when you pull. Make su
 
 ## Using Jupytext with the pre-commit package manager
 
-Using Jupytext with the [pre-commit package manager](https://pre-commit.com/) is another option. You could add the following to your `.pre-commit-config.yaml` file:
-```
+Using Jupytext with the [pre-commit package manager](https://pre-commit.com/) is another option. You could add the following to your `.pre-commit-config.yaml` file to convert all staged notebooks to python scripts in `py:percent` format (the default):
```

Comment:
There is a use case where the old way of doing it actually preferred, and that's when you want to run locally installed code. This commit hook is great for a general use case where conversion is the only part of JuPyText you want to use, but execution requires being in the local sandbox.

Reply:
I'm not sure I understand this comment. Why does it matter if it's installed locally or in the sandbox for this case?

Reply:
Requirements that can't be specified by pip install or are difficult to (like say the local master version of a pip package).